### PR TITLE
feat: remove XCM pallet from call filter

### PR DESCRIPTION
**Pull request by green-jay**

### CODE REVIEW

File: runtime/hydradx/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hydradx-runtime"
-version = "254.0.0"
+version = "255.0.1"
```
Comment (mrq1911) [inline runtime/hydradx/Cargo.toml:3]: patch bump as well ? :)

**Comment by mrq1911** [comment]
- [ ] can we wait for 1.11
- [ ] are the bemchmarks correct

**Comment by green-jay** [comment]
- snowfork might get resolved by moving DOT reserve in https://github.com/galacticcouncil/hydration-node/issues/857
- this should still be done to enable full functionality of the xcm pallet (transferAsset extrinsic etc)

File: integration-tests/src/call_filter.rs [hunk 2]
```diff
@@ -238,21 +238,6 @@ fn calling_pallet_xcm_send_extrinsic_should_not_be_filtered_by_call_filter() {
...
-	TestNet::reset();
-
-	Hydra::execute_with(|| {
-		// the values here don't need to make sense, all we need is a valid Call
-		let call = hydradx_runtime::RuntimeCall::PolkadotXcm(pallet_xcm::Call::execute {
-			message: Box::new(VersionedXcm::from(Xcm(vec![]))),
-			max_weight: Weight::zero(),
-		});
-
-		assert!(!hydradx_runtime::CallFilter::contains(&call));
```
Comment (dmoka) [inline integration-tests/src/call_filter.rs:252]: How about keeping the test, but flipping the boolean in the assertion?

**Comment by mrq1911** [comment]
- [ ] enable AH as reserve

**Comment by mrq1911** [comment]
blocked by #867

**Comment by green-jay** [comment]
superseeded by https://github.com/galacticcouncil/hydration-node/issues/895

**Pull request closed without merging**